pages/game.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import Link from "next/link";
+import { Container } from "react-bootstrap";
+
+export default function game() {
+    return (
+        <Container>
+            <h1>Hello, world!</h1>
+            <Link href="/about">About</Link>
+            <h1 class="text-centered">
+                ⬜⬜⬜⬜⬜ <br></br>
```

Comment:
**fitness for purpose**: I don't think the `<br>` tag needs a closing `</br>` tag - it seems that it's typically just used once without a closing tag. This would also clean up your html a little as well

Reply:
You can use the `<br />` tag, which is a self-closing break tag!